test/rules/validate-indentation.js

```diff
@@ -150,6 +150,67 @@ describe('rules/validate-indentation', function() {
         });
     });
 
+    describe('module pattern indentation', function() {
+        beforeEach(function() {
+            checker.configure({ validateIndentation: 4 });
+        });
+
+        var cases = {
+            'no indentation': 'a++;',
+            indentation: '    a++;'
+        };
+
+        Object.keys(cases).forEach(function(title) {
```

Comment:
These kind of test declarations are harder to isolate, not sure if copy/paste is worse, i did it couple times myself though